includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -13,7 +13,7 @@
  */
 class AMP_Allowed_Tags_Generated {
 
-	private static $spec_file_revision = 920;
+	private static $spec_file_revision = 935;
```

Comment:
Where does this come from? It does not seem to be the spec version this PR refers to ( `1908162134430` ).

This is probably part of the data that is being retrieved when generating this file, but a comment as to where this is mirrored might be useful.